### PR TITLE
CV Utils and Docs Improvements

### PR DESCRIPTION
**Pull request by zhanghang1989**
*Issue #, if available:*

*Description of changes:*
I am working on a challange and expand some relevant utils for CV in this PR.

1. Add RandomCrop Transformation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-169-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-169/1/index.html

**Comment by szha** [comment]
Job PR-169-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-169/2/index.html

File: autogluon/task/image_classification/image_classification.py [hunk 1]
```diff
@@ -100,8 +100,8 @@ def fit(dataset,
             training time limits in seconds.
         resources_per_trial : dict
             Machine resources to allocate per trial.
-        savedir : str
-            Local dir to save training results to.
+        output_directory : str
```
Comment (zhanghang1989) [inline autogluon/task/image_classification/image_classification.py]: @jwmueller Just renamed the variable to match the API name.

**Comment by szha** [comment]
Job PR-169-8 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-169/8/index.html

File: autogluon/utils/pil_transforms.py [hunk 2]
```diff
@@ -124,6 +129,96 @@ def resized_crop(img, top, left, height, width, size, interpolation=Image.BILINE
         img = resize(img, size, interpolation)
         return img
     
+class RandomCrop(object):
+    """Crop the given PIL Image at a random location.
+
+    Args:
```
Comment (Jerryzcn) [inline autogluon/utils/pil_transforms.py]: change it to parameters

File: autogluon/utils/pil_transforms.py [hunk 3]
```diff
@@ -124,6 +129,96 @@ def resized_crop(img, top, left, height, width, size, interpolation=Image.BILINE
...
+        self.padding = padding
+        self.pad_if_needed = pad_if_needed
+        self.fill = fill
+        self.padding_mode = padding_mode
+
+    @staticmethod
+    def get_params(img, output_size):
+        """Get parameters for ``crop`` for a random crop.
+
+        Args:
```
Comment (Jerryzcn) [inline autogluon/utils/pil_transforms.py]: ditto

**Pull request merged**